apps/react-app/src/pods/manfred-json-input/manfred-json-input.component.tsx

```diff
@@ -8,18 +8,17 @@ interface Props {
 export const ManfredJsonInput: React.FunctionComponent<Props> = (props: Props) => {
 
   const {onSetManfredJson} = props;
+  const {macData, setMacData} = React.useContext(MacContext);
 
-  const [text, setText] = useState('');
-
-  const handleChange = (event: { target: { value: React.SetStateAction<string>; }; }) => {
-    setText(event.target.value);
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setMacData(event.target.value);
   };
 
   const handleExport = () => {
 
-    if (text.trim().length !== 0) {
+    if (macData.trim().length !== 0) {
       alert('here will go the export to word');
```

Comment:
Remove this alert, now that we are navigating away to another page there's already a transition that will let the user notice something is going on